builtin/logical/pki/path_config_ca.go

```diff
@@ -52,6 +53,11 @@ func pathConfigIssuers(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: `Reference (name or identifier) to the default issuer.`,
 			},
+			"default_follows_latest_issuer": {
+				Type:        framework.TypeBool,
+				Description: `Whether the default issuer should automatically follow the latest generated or imported issuer. Defaults to false.`,
```

Comment:
If this is meant to allow opt-in to unbreak a change, should the original change be documented as breaking, and point to this new field as a way to opting into the previous behavior _if necessary_, while suggesting that callers move to using the new multi-issuer functionality more explicitly as they can? (just a thought)

Reply:
Ah yeah, its in the web docs that I've not pushed, one second.